src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+public class Storage {
+
+    /*
+     * reads file that stores the existing tasks and parses it.
```

Comment:
Javadocs start with a capital letter. Minor mistake.
```suggestion
     * Reads file that stores the existing tasks and parses it.
```